share/sql/share.go

```diff
@@ -341,59 +341,79 @@ func (m *shareMgr) UpdateReceivedShare(ctx context.Context, recvShare *collabora
 
 // Exported functions below are not part of the CS3-defined API, but are used by cernboxcop
 
-func (m *shareMgr) GetShareUnfiltered(ctx context.Context, ref *collaboration.ShareReference) (*model.Share, error) {
-	share, err := m.getShare(ctx, ref, false)
+func (m *ShareMgr) GetShareUnfiltered(ctx context.Context, ref *collaboration.ShareReference) (*model.Share, error) {
+	share, err := m.getShare(ctx, ref, false, false)
 	if err != nil {
 		return nil, err
 	}
 
 	return share, nil
 }
 
+func (m *ShareMgr) GetSharesByShareWith(ctx context.Context, shareWith string) ([]*collaboration.Share, error) {
```

Comment:
This is here but not used here, right? So please add a comment explaining. that this is an aux method, useful for external tools, e.g. cboxcop.

Reply:
Above there is this comment:

> // Exported functions below are not part of the CS3-defined API, but are used by cernboxcop

Reply:
Ah, it's above in the code...